manifests/server/install.pp

```diff
@@ -24,6 +24,12 @@
     }
   }
 
+  # Create Foreman share dir which does not depend on Puppet version
```

Comment:
Reminds me of https://github.com/theforeman/puppet-puppet/issues/67.

Reply:
which seems was later refactoring and then lost when introducing puppet 4 support